### DISCUSSION TITLE
BindablePropertyAttribute when will it be implemented?

### DISCUSSION THREAD

**Discussion by jnromeroYT** [CommunityToolkit/Maui#2372] [Q&A]
It seems curious to me that even though it has been in main for a long time but only as internal, when will they release it to the public?

**Comment by bijington** [comment]
The plan has been to dog feed it within the toolkit to make sure we are happy it works properly. I'm not sure when we will make it public, I'm not sure I've used it yet myself

**Comment by pictos** [comment]
Also, we need to figure out how to add inline docs for generated code, and that's, also, a good reminder that we (me) should update it and take advantage of partial properties. But since we're (I'm) slowing down due to the holidays you can expect any movement on that after january.
